### PR TITLE
Product settings

### PR DESCRIPTION
**Pull request by NimRegev**

### CODE REVIEW

**Review by nitronomad — APPROVED** [review]

File: _docs/products/configure-product-settings.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+After creating a product, configure metadata, Promotion Settings, and Promotion Flows for the Product and the applications connected to it through Product Settings. 
+
+This article describes how to configure the different Product Settings:
+
+* [Declaratively connect applications](#connect-applications-to-product-with-annotations)    
+  Declaratively connect an application to a product with the predefined and automatically generated product annotation. 
+  
+
+* [Labels for applications](#configure-labels)  
+  Add labels for the product by which to application.
```
Comment (oleksandr-codefresh) [inline _docs/products/configure-product-settings.md:17]: Add labels for the product (k8s metadata.labels)

File: _docs/products/configure-product-settings.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+
+
+## Connect applications to product with annotations
+Connect applications to products declaratively by copying the Product annotation to the application's manifest. If the Product doesn’t exist, Codefresh automatically creates it for you. (NIMA: is this correct)
+
+This is the preferred, declarative method of assigning applications to Products. The changes are committed and saved in Git as part of the application definition.    
+
+##### Before you begin
+* Review [Assigning applications to products]({{site.baseurl}}/docs/products/manage-products/assign-applications/)
```
Comment (oleksandr-codefresh) [inline _docs/products/configure-product-settings.md:60]: link not working

File: _docs/products/configure-product-settings.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+## Connect applications to product with annotations
+Connect applications to products declaratively by copying the Product annotation to the application's manifest. If the Product doesn’t exist, Codefresh automatically creates it for you. (NIMA: is this correct)
+
+This is the preferred, declarative method of assigning applications to Products. The changes are committed and saved in Git as part of the application definition.    
+
+##### Before you begin
+* Review [Assigning applications to products]({{site.baseurl}}/docs/products/manage-products/assign-applications/)   
+
+##### How to
+1. If needed, open [Product Settings](#open-product-settings).
```
Comment (oleksandr-codefresh) [inline _docs/products/configure-product-settings.md:63]: i guess we can skip `If needed`

File: _docs/products/configure-product-settings.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+    1. **File**: The name of the file from which to retrieve the version. For example, `chart.yaml`. 
+    1. **Path**: The JSON path to the attribute with the value. 
+    1. To see the result for any application connected to the product, click **Preview Configuration** and then select an application to see its version. 
+
+SCREENSHOT??
+
+{:start="5"}
+1. Define the settings for the **Promotable Properties**:
+    1. **File**: The name of the file with the attributes to promote. For example, `values.yaml`. 
+    1. **Paths**: The JSON path or paths to one or more attributes within the file to promote. For example, `$.buslog.image.tag`, `$.ctrlr.image.tag`, and `$.flask-ui.image.tag`.
```
Comment (oleksandr-codefresh) [inline _docs/products/configure-product-settings.md:177]: do you have some place where you provide more complex examples ?

File: _docs/products/promotion-version-properties.md [hunk 5]
```diff
@@ -0,0 +1,364 @@
...
+##### Primary functions
+Promotion Settings serve two primary functions for the applications being promoted:  
+1. Defining the source for the [application's release version](#version-for-promoted-applications)
+1. Defining the [changes to promote](#promotable-properties) across multiple files in the applications 
+
+
+{{site.data.callout.callout_important}}
+**IMPORTANT**  
+For automated retrieval of the application version and promotion of specific attributes from files across environments, _all applications within the same product must maintain an identical structure_.  
+Promotion across different environments requires consistent relative paths in each repository. For example, to promoyr proeprties from `config/settings.yaml` in environment `dev` to `testing`, `config/settings.yaml` must also exist in the `testing` environment.
```
Comment (oleksandr-codefresh) [inline _docs/products/promotion-version-properties.md:32]: `promoyr proeprties` typo

File: _docs/products/promotion-version-properties.md [hunk 6]
```diff
@@ -0,0 +1,364 @@
...
+
+
+For how-to instructions on configuring Promotion Settings, see [Configure Promotion Settings]({{site.baseurl}}docs/products/manage-products/configure-product-settings/#configure-promotion-settings).
+
+
+
+## Promotion Settings & Promotion Templates
+
+As with other GitOps entities, you can configure Promotion Settings in either Form or YAML modes.  
+Once configured and committed, these settings are saved as a CRD (Custom Resource Definition) entitled Promotion Template within the GitOps Runtime selected as the Configuration Runtime. This allows for a declarative and consistent approach to defining orchestration criteria across environments.
```
Comment (oleksandr-codefresh) [inline _docs/products/promotion-version-properties.md:43]: `GitOps Runtime selected as the Configuration Runtime` maybe provide som link to this info

File: _docs/products/promotion-flow-triggers.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+The Hotfix flow is designed for urgent fixes that need to be deployed quickly, typically to address critical bugs.
+
+
+
+ **Trigger condition** | **Setting** |
+|----------------------|------------------|
+| **Property**         | `commitMessage`  |
+| **Operator**          | `In`            |
+| **Values**           | `hotfix` `patch`  |
```
Comment (oleksandr-codefresh) [inline _docs/products/promotion-flow-triggers.md:57]: maybe better to split them into two line

File: _docs/products/configure-product-settings.md (hunk 2 above)
Comment (NimRegev) [reply _docs/products/configure-product-settings.md:60]: Fixed

File: _docs/products/configure-product-settings.md (hunk 3 above)
Comment (NimRegev) [reply _docs/products/configure-product-settings.md:63]: Agree

File: _docs/products/promotion-version-properties.md (hunk 6 above)
Comment (NimRegev) [reply _docs/products/promotion-version-properties.md:43]: Yes, also to ISC

**Pull request merged**